src/js/text-track-renderer/index.js

```diff
@@ -0,0 +1,137 @@
+import {
```

Comment:
this file would eventually land into `extras` or could be another project, like in open-elements or something.